src/tests/ftest/telemetry/telemetry_pool_space_metrics.py

```diff
@@ -0,0 +1,148 @@
+"""
```

Comment:
The name of the file is redundant. I'd recommend renaming
`telemetry/telemetry_pool_space_metrics.py`
to
`telemetry/pool_space_metrics.py`

Reply:
- [ ] Update the file name of the functional test